config.lua

```diff
@@ -1,431 +1,128 @@
-            coord = vector3(-513.06, -263.2, 35.43),
-            height = 17.6,
-            width = 4.2,
-            heading=293,
-            --debugPoly=true,
-            minZ=34.48,
-            maxZ=36.28
-        },
-        [14] = {
-            coord = vector3(-756.46, -35.84, 37.69),
-            height = 17.6,
-            width = 4.2,
-            heading=297,
-            --debugPoly=true,
-            minZ=36.74,
-            maxZ=38.54
-        },
-        [15] = {
-            coord = vector3(-1285.33, 293.67, 64.83),
-            height = 17.6,
-            width = 4.2,
-            heading=241,
-            --debugPoly=true,
-            minZ=63.88,
-            maxZ=65.68
-        },
-        [16] = {
-            coord = vector3(-806.68, 825.2, 202.81),
-            height = 21.2,
-            width = 4.2,
-            heading=276,
-            --debugPoly=true,
-            minZ=200.46,
-            maxZ=204.66
-        },
+        [1] = {coord = vec3(258.98, -377.9, 44.7), height = 17.6, width = 4.2, heading = 69, minZ = 43.75, maxZ = 45.55},
+        [2] = {coord = vec3(-50.06, -784.57, 44.16), height = 17.6, width = 4.2, heading = 62, minZ = 43.21, maxZ = 45.01},
+        [3] = {coord = vec3(238.93, -858.91, 29.67), height = 17.6, width = 4.2, heading = 71, minZ = 28.72, maxZ = 30.52},
+        [4] = {coord = vec3(823.4, -1882.96, 29.29), height = 17.6, width = 4.2, heading = 167, minZ = 28.34, maxZ = 30.14},
+        [5] = {coord = vec3(354.05, -1971.57, 24.43), height = 17.6, width = 4.2, heading = 236, minZ = 23.48, maxZ = 25.28},
+        [6] = {coord = vec3(-225.61, -2043.63, 27.62), height = 17.6, width = 4.2, heading = 143, minZ = 26.67,  maxZ = 28.47},
+        [7] = {coord = vec3(-1048.72, -2714.2, 13.76), height = 17.6, width = 4.2, heading = 240, minZ = 12.81, maxZ = 14.61},
+        [8] = {oord = vec3(-776.15, -1280.37, 5.0), height = 17.6, width = 4.2, heading = 261, minZ = 4.05, maxZ = 5.85},
+        [9] = {coord = vec3(-1180.3, -1304.22, 5.15), height = 17.6, width = 4.2, heading = 205, minZ = 4.2, maxZ = 6.0},
+        [10] = {coord = vec3(-1326.52, -833.32, 16.85), height = 17.6, width = 4.2, heading = 225, minZ = 15.9,  maxZ = 17.7},
+        [11] = {coord = vec3(-1610.24, -1015.33, 13.07), height = 17.6, width = 4.2, heading = 227, minZ = 12.12, maxZ = 13.92},
+        [12] = {coord = vec3(-1396.85, -583.72, 30.08), height = 17.6, width = 4.2, heading = 299, minZ = 29.13, maxZ = 30.93},
+        [13] = {coord = vec3(-513.06, -263.2, 35.43), height = 17.6,  width = 4.2, heading = 293, minZ = 34.48,  maxZ = 36.28},
+        [14] = { coord = vec3(-756.46, -35.84, 37.69), height = 17.6, width = 4.2, heading = 297, minZ = 36.74, maxZ = 38.54},
+        [15] = {coord = vec3(-1285.33, 293.67, 64.83), height = 17.6, width = 4.2, heading = 241, minZ = 63.88, maxZ = 65.68},
+        [16] = {coord = vec3(-806.68, 825.2, 202.81), height = 21.2, width = 4.2, heading = 276, minZ = 200.46, maxZ = 204.66}
```

Comment:
It's great to see refactoring but while doing so please remove the [index] = vec shit.

In a standard lua array its always gonna be 1, 2, 3, 4 etc. you dont need to define that shit, only if u want a specific ID thats not in a chronological order.

Reply:
I plan to. This is just formatting/style changes. Actual, working changes are next. I just didnt wanna do a massive PR